po/fr.po

```diff
@@ -0,0 +1,876 @@
+# quick-lint-js finds bugs in JavaScript programs.
```

Comment:
Is this Canadian French or France French?

Canadian French: Call this `fr_CA.po`
France French: Call this `fr_FR.po`

Reply:
France French. I renamed the translation file as suggested above.